gadgets/top_file/README.mdx

```diff
@@ -0,0 +1,176 @@
+---
+title: top_file
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# top_file
+
+The top_file gadget reports periodically the read/write activity by file.
+
+## Getting started
+
+Running the gadget:
+
+<Tabs groupId="env">
+    <TabItem value="kubectl-gadget" label="kubectl gadget">
+        ```bash
+        $ kubectl gadget run ghcr.io/inspektor-gadget/gadget/top_file:latest [flags]
+        ```
+    </TabItem>
+
+    <TabItem value="ig" label="ig">
+        ```bash
+        $ ig run ghcr.io/inspektor-gadget/gadget/top_file:latest [flags]
+        ```
+    </TabItem>
+</Tabs>
+
+## Flags
+
+TODO: Add flags added by operators.
+
+### `--all-files`
+Trace all files types. By default, only regular files are traced
```

Comment:
nit: add space before `###` (same in other places)

Reply:
I guess you after 🙂